### PR TITLE
Feat: contract info endpoint

### PR DESCRIPTION
**Pull request by schmanu**
Adds contract info endpoint which fetched contract name, logo and abis.

### CODE REVIEW

**Review by katspaugh — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@safe-global/safe-gateway-typescript-sdk",
-  "version": "3.20.0",
+  "version": "3.21.0",
```
Comment (katspaugh) [inline package.json:3]: Next time let's do it as a separate commit, easier to see in the history when a version was released.

Comment (schmanu) [reply package.json:3]: I can do a normal merge (non-squash) to keep it a separate commit :)

Comment (schmanu) [reply package.json:3]: Ah nvm its disabled.
